json_schemas/services-g5.json

```diff
@@ -66,7 +66,7 @@
     },
     "minimumContractPeriod": {
       "type": "string",
-      "pattern": "^(hour|day|month|year|other)$"
+      "pattern": "^([Hh]our|[Dd]ay|[Mm]onth|[Yy]ear|[Oo]ther)$"
```

Comment:
We should rewrite the patterns as enums following #130, or make sure this gets merged first.